spec/unit/chitter_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "peep"
+
+describe Peep do
+  describe ".create" do
+    it "creates a new peep" do
+      connection = PG.connect(dbname: "chitter_peeps_test")
```

Comment:
This line doesn't do anything, as you don't refer to this `connection` variable in the `it` block.